extension-manifest-v3/src/utils/trackerdb.js

```diff
@@ -94,11 +99,12 @@ export async function getPattern(key) {
 
 export function isCategoryBlockedByDefault(categoryId) {
```

Comment:
Are you sure that only 4 categories are blocked by default?

Reply:
yups, I'm positive